grails-app/services/com/unifina/service/EthereumIntegrationKeyService.groovy

```diff
@@ -31,18 +34,40 @@ class EthereumIntegrationKeyService {
 
 	@PostConstruct
 	void init() {
```

Comment:
Now with the lazy creation of `encryptor`, I guess the whole `init()` function can be removed?